### DISCUSSION TITLE
Possible to include broadcomm WiFi drivers for bare metal mac installation

### DISCUSSION THREAD

**Discussion by Deep-synth** [dat-linux/community#38] [Issues]
As said. Installed dat Linux and was a disappointed it didn't have Broadcom WiFi drivers is it possible to maybe include them in the distro?

**Comment by dat-linux** [comment]
The installation process for each use case would require research and installation to suit. See https://devicetests.com/install-broadcom-wireless-drivers-ubuntu

It's not viable to "just install drivers" in the distro as you say. DAT Linux packages whatever Lubuntu packages (which is a Ubuntu official spin), and Linux is open to freely post-install whatever you need.
